operations/pyroscope/helm/pyroscope/values.yaml

```diff
@@ -2,6 +2,9 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
+# Override default K8S cluster domain for DNS discovery
+# cluster_domain:
```

Comment:
I think it should life within pyroscope, we can argue that everything within `pyroscope`, should be global yes, but that is now very hard to change.

Also this should set the default:

```suggestion
# Override default K8S cluster domain for DNS discovery
cluster_domain: ".cluster.local."
```